Airport Ticket Booking/Passenger.cs

```diff
@@ -0,0 +1,272 @@
+                            MaxPrice = UserInput((int)IntOrDouble.doublen);
+                            break;
+                        default:
+                            break;
+                    }
+                }
+                List<Flight> FilterFlights = FlightsManager.SearchFlights(DepartureCountry, DestinationCountry, DepartureAirport, ArrivalAirport, DepartureDate, FlightClass, MaxPrice);
+                FlightsManager.showFlights(FilterFlights);
+            }
+            else
+            {
+                Console.WriteLine("The Input Is Not Contains Any Options");
+            }
+
+        }
+
+        private void CancelAskInput()
+        {
+            Console.WriteLine("Please Enter The Id of the Booking you Want to Cancel: ");
+            int id = (int)UserInput((int)IntOrDouble.integern);
+            if (!FlightsManager.IsThereBookingWithThisId(id))
+            {
+                Console.WriteLine("There is no Booking with this Id!");
+            }
+            else
+            { 
+                FlightsManager.CancelBooking(id);
+                Console.WriteLine("Cancelled Successfully");
+            }
+        }
+
+        private void ViewAskInput()
+        {
+            Console.WriteLine("Please Enter The Name that You Booked Using It");
+            string Name = Console.ReadLine();
+            List<Booking> BookingsName = FlightsManager.ShowBooking(Name);
+            if(BookingsName == null)
+            {
+                Console.WriteLine("There is no booking assigned to this name!");
+            }
+            foreach(Booking bookedFlight in BookingsName)
+            {
+                Console.WriteLine($"Id: {bookedFlight.Id}, Passenger Name: {bookedFlight.PassengerName}, " +
+               $"Flight Calss: {bookedFlight.FClass}, Flight Information: ");
+                Console.WriteLine($"{bookedFlight.Flight}");
+            }
+        }
+
+        private void ModifyAskInput()
+        {
+            Console.WriteLine("Please Enter The Id of the Booking that you want to modify: ");
+            int id = (int)UserInput((int)IntOrDouble.integern);
+            bool IsBooking = FlightsManager.IsThereBookingWithThisId(id);
+            if (IsBooking)
+            {
+                Console.WriteLine("Choose from the following what do you want to modify: ");
+                Console.WriteLine("1. Modify the Passenger Name");
+                Console.WriteLine("2. Modify the Flight Id");
+                Console.WriteLine("3. Modify The Class");
+                Console.WriteLine("4. Modify all");
+
+                int number = (int)UserInput((int)IntOrDouble.integern, 1, 4);
+                string newPassName = null;
+                int NewflightId = 1; // By the end the id will change it is just initial value
+                int NewFlightClass = (int) 1;
+                switch (number)
+                {
+                    case 1:
```

Comment:
I see that you're overusing switch cases in this project how about trying to implement a table-based solution? something like a dictionary?